e2e/apiserver/apiserver_test.go

```diff
@@ -21,35 +19,39 @@ import (
 )
 
 var (
-	envHelloMeta = types.EnvMeta{
-		Name:      "env-e2e-hello",
+	envHelloMeta = apis.Environment{
```

Comment:
What's the purpose of this change?

Reply:
Environment object provides input parameter verification of api interface

Reply:
types.EnvMeta could not?

Reply:
types.EnvMeta It's domain entity. Environment is similar to dto,in the future, the interface can carry more expressive force